frameworks/cassandra/docs/limitations.md

```diff
@@ -5,11 +5,23 @@ feature_maturity: preview
 enterprise: 'no'
 ---
 
-* Data Center Name
+## Rack-Aware Placement
+
+Apache Cassandra's Rack-Aware Replication is not supported at this time.
+
+## Security
+
+Apache Cassandra's native TLS, authentication, and authorization features are not supported at this time.
+
+## Overlay networks
+
+When the DC/OS Cassandra Service is deployed on a virtual network, the configuration cannot be updated after installation.
```

Comment:
The configuration here meaning which overlay network it belongs to, correct? That could be clarified.